spec/plugins/aws-lambda/99-access_spec.lua

```diff
@@ -8,7 +8,6 @@ local TEST_CONF = helpers.test_conf
 local server_tokens = meta._SERVER_TOKENS
 local null = ngx.null
 
-
```

Comment:
please do not remove this. We ❤️ vertical white space at Kong 😄 , makes things so much easier on the 👀 .

Reply:
I'll add this whitespace back